Scyther Playground/Scyther Playground.xcodeproj/project.pbxproj

```diff
@@ -631,20 +631,20 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		CA73FDDB25DA22E8007FA627 /* XCRemoteSwiftPackageReference "Scyther" */ = {
+		C9A3A2362666BAD400E2B2F9 /* XCRemoteSwiftPackageReference "Scyther" */ = {
 			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/bstillitano/Scyther.git";
+			repositoryURL = "file:///Users/stefanhaider/Documents/XcodeWorkespace/Scyther";
```

Comment:
Please revert this change.

Reply:
i already changed that the playground uses the local swift package so someone can play around locally and experiment a little, if this is a problem i'll change it back to the original implementation.

Reply:
The playground is now pointing to your local folder on your machine. This will not work on anyone else's machine but yours.